internal/pkg/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -352,7 +352,14 @@ func (u *ALMExamplesUpdate) Apply(csv *olmapiv1alpha1.ClusterServiceVersion) err
 		}
 	}
 	sb.WriteString(`]`)
-
-	csv.GetAnnotations()["alm-examples"] = sb.String()
+	format, _ := prettyPrint([]byte(sb.String()))
```

Comment:
Return this error if non-nil

Reply:
Also, I'd suggest a more descriptive variable name, perhaps `prettyJSON` to go along with @estroz's function name suggestion. WDYT?

Reply:
Done tks for the review 👍